nautobot/docs/release-notes/version-1.3.md

```diff
@@ -56,7 +56,11 @@ The expressions `re` (regex), `nre` (negated regex), `ire` (case-insensitive reg
 
 Nautobot's REST API now supports multiple versions, which may requested by modifying the HTTP Accept header on any requests sent by a REST API client. Details are in the [REST API documentation](../rest-api/overview.md#versioning), but in brief:
 
-- The only REST API endpoint that is versioned in the 1.3.0 release is the `/api/extras/jobs/` listing endpoint, as described above. All others are currently un-versioned. However, over time more versioned REST APIs will be developed, so this is important to understand for all REST API consumers.
+- The REST API endpoints that are versioned in the 1.3.0 release are
+  - `/api/extras/jobs/` listing endpoint
+  - `/api/extras/tags/` create/put/patch endpoint
+  - `/api/ipam/ip-addresses/` endpoints
+- All others endpoints are currently un-versioned. However, over time more versioned REST APIs will be developed, so this is important to understand for all REST API consumers.
```

Comment:
For some reason, ReadTheDocs has a different expectation on indenting of nested lists than local rendering with `mkdocs` alone - the below is needed for correct rendering on RTD. See also #1557.

```suggestion
- The REST API endpoints that are versioned in the 1.3.0 release are
    - `/api/extras/jobs/` listing endpoint
    - `/api/extras/tags/` create/put/patch endpoint
    - `/api/ipam/ip-addresses/` endpoints
- All others endpoints are currently un-versioned. However, over time more versioned REST APIs will be developed, so this is important to understand for all REST API consumers.
```